pom.xml

```
@@ -104,13 +104,22 @@
                             <goal>copy-resources</goal>
                         </goals>
                         <configuration>
-                          <outputDirectory>${basedir}/docs</outputDirectory>
-                          <resources>          
-                              <resource>
-                                  <directory>target/site</directory>
-                              </resource>
-                          </resources>              
-                        </configuration>            
+                            <outputDirectory>${basedir}/docs</outputDirectory>
+                            <nonFilteredFileExtensions>
+                                <nonFilteredFileExtension>pdf</nonFilteredFileExtension>
+                                <nonFilteredFileExtension>jar</nonFilteredFileExtension>
+                                <nonFilteredFileExtension>html</nonFilteredFileExtension>
+                            </nonFilteredFileExtensions>
+                            <resources>
+                                <resource>
+                                    <directory>target/site</directory>
+                                </resource>
```

The output directory should be the configured build directory as the deploy action just looks at the resources located in `target/site`.

Additionally, the previous configuration moving resources from `target/site` is no longer necessary. It was previously used to move the build results into the `docs` directory.

And, as a minor change, I don't think the filter exceptions are necessary as filtering is explicitly disabled for the resource.

```suggestion
                            <outputDirectory>${build.dir}</outputDirectory>
                            <resources>
```

---

Thanks for reviewing, I cleaned it up a bit. Looks better now :-)